internal/notification/projection.go

```diff
@@ -42,6 +42,7 @@ func Start(ctx context.Context, customConfig projection.CustomConfig, externalPo
 }
 
 type notificationsProjection struct {
+	ctx func(event eventstore.Aggregate) context.Context
```

Comment:
@livio-a because you don't like embedding the context directly into the struct :joy: